charts/keycloak/values.yaml

```diff
@@ -18,6 +18,22 @@ keycloak:
       standardFlowEnabled: true
       webOrigins:
       - +
+    - clientId: sigstore
+      enabled: true
+      name: Sigstore Client (RHTAS)
+      protocol: openid-connect
+      publicClient: true
+      redirectUris:
+      - '*'
```

Comment:
This is ok for now. but we should review this in the future to further lock down redirect URI's